.github/workflows/OneShedule.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and deploy .NET Core app to Linux WebApp OneShedule
+on:
+  push:
+    branches:
+    - feature/deployed-fixed
+    - develop
+env:
+  AZURE_WEBAPP_NAME: OneShedule
+  AZURE_WEBAPP_PACKAGE_PATH: OneSchedule/publish
+  AZURE_WEBAPP_PUBLISH_PROFILE: ${{ secrets.OneShedule_0db7 }}
+  TelegramSettings.ApiKey: ${{ secrets.TELEGRAMSETTINGS_APIKEY }} 
+  TelegramSettings.TestUri: ${{ secrets.TELEGRAMSETTINGS_TESTURI }} 
+  DatabaseSettings.ConnectionString: ${{ secrets.DATABASESETTINGS_CONNECTIONSTRING }} 
+  DatabaseSettings.DatabaseName: ${{ secrets.DATABASESETTINGS_DATABASENAME }} 
+  CONFIGURATION: Release
+  DOTNET_CORE_VERSION: 5.0.x
+  WORKING_DIRECTORY: OneSchedule
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ env.DOTNET_CORE_VERSION }}
+    - name: Restore 
+      run: dotnet restore "${{ env.WORKING_DIRECTORY }}" 
+    - name: App Settings Variable Substitution 
```

Comment:
Substitution can be done right before the _publish_ or even after because those variables are used only during the app launch